test/run_test_apache.sh

```diff
@@ -175,6 +175,7 @@ if [ $testError -ne 0 ]; then
     RED='\033[0;31m'
     NC='\033[0m' # No Color
     echo -e "${RED} There is error above this line ${NC}"
-    cat $APACHE_LOG_PATH/kc.log
+    #    Ignore for e2e test draft. Todo: turn back again?
+    #    cat $APACHE_LOG_PATH/kc.log
```

Comment:
It's horrible to check what has failed when printing **all** Kafka connector debug logs. Once all tests have passed, this change will be reverted back.